src/interfaces/coral_web/src/components/Agents/AgentForm.tsx

```diff
@@ -5,7 +5,10 @@ import { Checkbox, Input, InputLabel, STYLE_LEVEL_TO_CLASSES, Text } from '@/com
 import { useListTools } from '@/hooks/tools';
 import { cn } from '@/utils';
 
-export type AgentFormFields = Omit<CreateAgent, 'version' | 'temperature'>;
+export type AgentFormFields = Pick<
+  CreateAgent,
+  'name' | 'description' | 'preamble' | 'deployment' | 'model' | 'tools'
+>;
```

Comment:
a bit more future-proof since i was getting an error about `tools_metadata`, which is a new field